k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml

```diff
@@ -0,0 +1,21 @@
+# this is to apply permanent redirect on www-ccd.perftest.platform.hmcts.net
+---
+apiVersion: extensions/v1beta1
```

Comment:
could you update the latest ingress api? this is deprecated for removal

Reply:
cheers - now updated